packages/image_picker/image_picker_for_web/example/integration_test/image_resizer_test.dart

```diff
@@ -136,5 +135,5 @@ web.Blob _base64ToBlob(String data) {
     n--;
   }
 
-  return Blob(<JSUint8Array>[u8arr.toJS].toJS);
+  return web.Blob(<JSUint8Array>[u8arr.toJS].toJS);
```

Comment:
It looks like this is in `0.5.1`; do we actually need to require 1.0.0 to make this change?